<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-50</y>
      <width>1080</width>
      <height>1080</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>154</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Uložit</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>156</width>
      <height>175</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="prekazkaLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>132</height>
        </size>
       </property>
       <property name="text">
        <string>Překážka</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xObstacle">
       <property name="text">
        <string>x souřadnice</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yObstacle">
       <property name="text">
        <string>y souřadnice</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="obsSize">
       <property name="text">
        <string>velikost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="obsRotation">
       <property name="text">
        <string>rotace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addObstacle">
       <property name="text">
        <string>Přidat překážku</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="playPauseButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>158</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>158</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="RobotVerical">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Robot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xLineEdit">
       <property name="text">
        <string>x souřadnice</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yLineEdit">
       <property name="text">
        <string>y souřadnice</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="angleLineEdit">
       <property name="text">
        <string>úhel - start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="speedLineEdit">
       <property name="text">
        <string>rychlost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="turnAngleLineEdit">
       <property name="text">
        <string>úhel - otočení</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rotationDirectionLineEdit">
       <property name="text">
        <string>Směr (Left, Right)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="colorLineEdit">
       <property name="text">
        <string>Barva</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="detectionDistanceLineEdit">
       <property name="text">
        <string>Detekce vzdálenosti</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addRobotButton">
       <property name="text">
        <string>Přidat robota</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
